app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        person_list.append(Person(name, age))
+
+    for person in people:
+        if "wife" in person and person["wife"] in Person.people:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        if "husband" in person and person["husband"] in Person.people:
+            Person.people[person["name"]].husband = Person.people[
+                person["husband"]
+            ]
```

Comment:
Think about creating additional variable for Person.people[person["name"]], this will make your code look cleaner